File: w6_python/phonebook.py
# Implements a phone book
from cs50 import get_string

# examples
scores = list()
scores2 = [] # same

peoplee = dict()
peoplee2 = {} # same

people = {                          # dictionary with key-value pairs
    "Carter": "+1-617-495-1000",
    "David": "+1-949-468-2750"
}

# Search for name
name = get_string("Name: ")
if name in people:  # search key
    print(f"Number: {people[name]}") # get the value people[name]

File: w3_algorithms/phonebook.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>

int main(void)
{
    string names[] = {"Carter", "David" };
    string numbers[] = {"0910-123-456", "0918-456-878" };

    string name = get_string("Name: ");
    for (int i = 0; i < 2; i++)
    {
        if (strcmp(names[i],name) == 0)
        {
            printf("Found the number is %s\n", numbers[i]);
            return 0;
        }
    }
        printf("Not found\n");
        return 1;
}